build-docker-image/action.yml

```diff
@@ -79,6 +82,7 @@ runs:
     uses: docker/build-push-action@v3
     with:
       context: .
+      build-args: ${{ inputs.build_args }}
```

Comment:
Since `inputs.build_args` is optional did you check what happens if it is not set? Does everything still work and it is assumed to be empty string orso

Reply:
Good question. It wil skip over/ignore it if it's not set. Confirmed in this [build](https://github.com/camunda/camunda-docker-ci-chrome/actions/runs/4005644476/jobs/6876251749) by setting it to build against this branch but without passing the _build-args_ to the action.